src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailedPackageMetadata.cs

```diff
@@ -51,7 +49,7 @@ public DetailedPackageMetadata()
             LicenseMetadata = serverData.LicenseMetadata;
             DeprecationMetadata = deprecationMetadata;
             Vulnerabilities = serverData.Vulnerabilities;
-            _localMetadata = serverData as LocalPackageSearchMetadata;
+            _localMetadata = null; //TODO: SCOBAN // serverData as LocalPackageSearchMetadata;
```

Comment:
you already know about the two //TODO: SCOBAN in the PR. Tracking with this comment too.